src/UserBundle/Maker/UserMaker.php

```diff
@@ -433,6 +433,10 @@ private function generateControllers(ConsoleStyle $io): void
             ])];
         }
 
+        if ($hasLogin && $io->confirm('Activate /logout route (Add config/routes.yaml)?')) {
```

Comment:
we need to account for this question in generated security.yaml (in case the answer is "no" we should not add the logout section).

so lets ask this first, before we ask about security.yaml

Reply:
also i'd like to ask for `Can users logout? (adds config/routes.yaml)`